### PR TITLE
AddGarrison - Improve

### PR DESCRIPTION
**Pull request by StefArma**
1) Assign an AI recruited/controlled  group to Garrison
2) Assign a vehicle as garrison: place the vehicle in game, and save its position for the garrison.

### CODE REVIEW

File: AntistasiOfficial.Altis/REINF/addgarrison_improve.sqf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    //input checks
+    if (_pos distance _marker > 50) exitWith  {hint "Bring units closer to the emplacement"};
+      
+    //add unit-type to garrison and delete previous units
+    
+    
+    
+//2) Assign a vehicle as garrison
+    //input
+    _veh = cursorobject //using Y menu
```
Comment (Barbolani77) [inline AntistasiOfficial.Altis/REINF/addgarrison_improve.sqf]: typo

Comment (StefArma) [reply AntistasiOfficial.Altis/REINF/addgarrison_improve.sqf]: hi i see

**Comment by StefArma** [comment]
Barbolani did the first part:
https://github.com/A3Antistasi/antistasiofficial/commit/692762a6705a0fe85092ce0437edd38968c6a0a3

**Comment by Barbolani77** [comment]
Assigning a vehicle for a garrison is a matter of two possibilities:

Add it to the staticsToSave array. The problema is those vehicles will never despawn. The advantage: vehicle will be in a right position, as it has been left there by humans.

Modify the spawning script so it accepts vehicle classnames, the problem here is the vehicle will spawn in whatever empty position with the little risk of explosion + senseless positions. Thats why I oncly dode mortars because as long as you dont spawn them in a place with roof, they will be operative.

**Pull request closed without merging**